CHANGELOG.md

```diff
@@ -30,6 +30,10 @@
 ### Early Access Features
 - Enable plugins to expose custom JSON-RPC / WebSocket methods [#1317](https://github.com/hyperledger/besu/issues/1317)
 
+### Download Link
+https://hyperledger.jfrog.io/ui/native/besu-binaries/besu/21.10.1/besu-21.10.1.zip \
+SHA256: 877148a1a13a1065151cf8fb3fbb56ea1d01e3a942fa7ebd6bd4ea4d15101fee
```

Comment:
I am getting a different SHA... 
shasum -a 256 ~/Downloads/besu-21.10.1.zip
064df81be49c31ebfc0e1a5f03357e8f6f9a7f5e7bcb82e2b187e2d58e1704eb

Reply:
Gary's matches the homebrew hash - https://github.com/hyperledger/homebrew-besu/blob/main/besu.rb#L6

Reply:
I had downloaded the native zip. fixed.

Reply:
yep I had the wrong link. fixed